clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -178,4 +184,66 @@ public static Properties producerConfig(final String bootstrapServers,
         properties.putAll(additional);
         return properties;
     }
+
+    // provides producer properties with default String serializers for keys and values
+    public static Properties producerConfigStringKeysValues(final String bootstrapServers) {
```

Comment:
Could we just add a `producerConfig` function that takes three parameters, for bootstrapServers and key-value serdes?